guacamole/src/main/frontend/src/app/clipboard/directives/guacClipboard.js

```diff
@@ -71,6 +89,32 @@ angular.module('clipboard').directive('guacClipboard', ['$injector',
 
         };
 
+        /**
+         * Allows Clipboard Editor to begin displaying clipboard data, reads the
+         * clipboard data and updates the content of the clipboard editor to the
+         * given data.
+         */
+        var displayAndUpdateClipboardEditor = function displayAndUpdateClipboardEditor() { 
+            isActive = true;
+            element.value = ''
+            element.classList.remove("inactive");
+
+            clipboardService.getClipboard().then((data) => {
+                updateClipboardEditor(data);
+            }, angular.noop);
+        };
+
+        /**
+         * Prevent the Clipboard Editor from displaying clipboard data.
+         */
+        var hideAndUpdateClipboardEditor = function hideAndUpdateClipboardEditor() { 
```

Comment:
This entire function probably doesn't need to exist - all this can be done directly in the template with conditional rendering depending on the value of `isActive`.